### PR TITLE
Digital Credentials: digital-credentials-get Permissions Policy tests

### PR DESCRIPTION
**Pull request by marcoscaceres**
Tests for https://github.com/WICG/digital-credentials/pull/132

### CODE REVIEW

File: digital-credentials/default-permissions-policy.https.sub.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<body></body>
+<script>
+    "use strict";
+    const { HTTPS_REMOTE_ORIGIN } = get_host_info();
+    const same_origin_src =
+        "/permissions-policy/resources/digital-credentials-get.html";
+    const cross_origin_src = new URL(same_origin_src, HTTPS_REMOTE_ORIGIN).href;
+
+    promise_test(async (test) => {
+        await test_driver.bless("use activation");
```
Comment (pkotwicz) [inline digital-credentials/default-permissions-policy.https.sub.html:18]: Nit: use->user

File: digital-credentials/override-permissions-policy.https.sub.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/permissions-policy/resources/permissions-policy.js"></script>
+<script src="/common/get-host-info.sub.js"></script>
+<body></body>
+<script>
```
Comment (pkotwicz) [inline digital-credentials/override-permissions-policy.https.sub.html:10]: What is the difference between this test and enabled-on-self-origin-by-permissions-policy.https.sub.html ?

File: digital-credentials/get-user-activation.https.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```
Comment (pkotwicz) [inline digital-credentials/get-user-activation.https.html:1]: Is this a bad merge?

Isn't this file added by https://github.com/web-platform-tests/wpt/pull/46642

File: digital-credentials/enabled-on-self-origin-by-permissions-policy.https.sub.html.headers [hunk 4]
```diff
@@ -0,0 +1 @@
+Permissions-Policy: digital-credentials-get=(self)
```
Comment (pkotwicz) [inline digital-credentials/enabled-on-self-origin-by-permissions-policy.https.sub.html.headers:1]: Optional: Add test for
Permissions-Policy: digital-credentials-get=*

**Review by pkotwicz2 — CHANGES_REQUESTED** [review]
This CL looks mostly good. A few questions.

File: digital-credentials/allow-attribute.https.html [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+                    isAllowed: true,
+                },
+                {
+                    policy: `digital-credentials-get ${hostInfo.HTTPS_REMOTE_ORIGIN}`,
+                    crossOrigin: false,
+                    isAllowed: false,
+                },
+            ];
+
+            async function loadIframe({ policy, crossOrigin, isAllowed }) {
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply digital-credentials/allow-attribute.https.html:76]: Good point. Will fix!

File: digital-credentials/get-user-activation.https.html (hunk 3 above)
Comment (marcoscaceres) [reply digital-credentials/get-user-activation.https.html:1]: oh yes, sorry... I copied too many files over. 🙈

File: digital-credentials/override-permissions-policy.https.sub.html (hunk 2 above)
Comment (marcoscaceres) [reply digital-credentials/override-permissions-policy.https.sub.html:10]: The intent here was to check explicitly overriding the policy through the allow attribute. Here, the policy is disabled by the HTTP header on the top-level.

🤔 ... Now I'm wondering if maybe these should move to digital-credentials/disabled-by-permissions-policy.https.sub.html ... though I'm ok to leave them here.

File: digital-credentials/enabled-on-self-origin-by-permissions-policy.https.sub.html.headers (hunk 4 above)
Comment (marcoscaceres) [reply digital-credentials/enabled-on-self-origin-by-permissions-policy.https.sub.html.headers:1]: I might leave this one for now and do it as a followup. I need to implement support for `Permissions-Policy:` generally in WebKit still, so will need to do test it as part of that.

**Review by pkotwicz2 — APPROVED** [review]
Approve with request to move override-permission-policy tests

File: digital-credentials/override-permissions-policy.https.sub.html (hunk 2 above)
Comment (marcoscaceres) [reply digital-credentials/override-permissions-policy.https.sub.html:10]: Yeah, same about consistency. Thanks for spotting this! 

I'll fix that up now.

**Review by annevk — APPROVED** [review]
Rubberstamp per earlier review.

**Pull request merged**